openstack/manila/templates/seeds.yaml

```diff
@@ -64,6 +64,8 @@ spec:
   - name: cloud_sharedfilesystem_viewer
   - name: sharedfilesystem_admin
   - name: sharedfilesystem_viewer
+{{- Role `cloud_objectstore_admin` needed for backup to cloud service }}
+  - name: cloud_objectstore_admin
```

Comment:
This is seeded in `swift-seed`, so maybe we should add a dependency to that above under `spec.requires`.
But this line here should be removed